### ISSUE TITLE
xperia play down key still not working on v1.7 and still no config to define keys as in v1.5...please fix cos v1.5 worked better.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ionux/genplus-droid#4]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.


```

Original issue reported on code.google.com by `m.asquit...@gmail.com` on 16 Aug 2011 at 12:25

**Comment by GoogleCodeExporter** [comment]
```
The default key bindings are for Xperia Play.  There have been a few people who 
for some reason or another get this bug.  You can try hitting the "Reset Input" 
in Settings.  If that fails usually just uninstall / reinstall should fix it.  
The defaults are for Xperia Play.
```

Original comment by `shinhals...@gmail.com` on 19 Sep 2011 at 3:59
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed as of version 1.8.  Please let me know.
```

Original comment by `shinhals...@gmail.com` on 28 Sep 2011 at 5:04
- Changed state: **Fixed**
